rollup/tests/RollupFlowBulkProcessorTests.cls

```diff
@@ -17,9 +22,11 @@ private class RollupFlowBulkProcessorTests {
 
     List<Rollup.FlowOutput> outputs = RollupFlowBulkProcessor.addRollup(new List<RollupFlowBulkProcessor.FlowInput>{ input });
 
-    System.assertEquals(true, outputs[0].message.contains('SUM'));
-    System.assertEquals(true, outputs[0].isSuccess, 'Should not error when adding deferred flow rollup');
-    System.assertEquals(1, Rollup.FLOW_ROLLUPS.size(), 'Deferred rollup should be kept in buffer');
+    System.assertNotEquals(true, outputs.isEmpty(), 'Should have passed back data from inputs');
```

Comment:
I ended up reshuffling some of the asserts here because this test failed due to the actual metadata I had in my scratch org to test the CMDT-based full recalculation; I thought I’d make it less flaky, as a result